controllers/app.controller.js

```diff
@@ -44,28 +44,35 @@ exports.postCommentToArticle = (req, res, next) => {
     if(!username || !body){
         return res.status(400).send({msg: "Bad request: missing username or body"});
     }
-    Promise.all([checkUserExists(username), checkArticleExist(article_id)]).then(() => {
+    Promise.all([checkUserExists(username), fetchArticleById(article_id)]).then(() => {
         return addComment(article_id, username, body);
     }).then((comment) => {
         res.status(201).send({comment})
     })
-    .catch(next)
+    .catch(next);
 }
 
 exports.patchArticleVotes = (req, res, next) => {
     const { article_id } = req.params;
     const { inc_votes } = req.body;
 
-    Promise.all([checkArticleExist(article_id)]).then(() => {
+    Promise.all([fetchArticleById (article_id)]).then(() => {
         return updateVotes(article_id, inc_votes);
     })
    .then((updatedArticle) => {
         res.status(200).send({ article: updatedArticle })
     })
-    .catch((err) => {
-        console.log(err)
-        next(err)
-    })
+    .catch(next);
 }
 
-
+exports.deleteCommentById = (req, res, next) => {
+    const { comment_id } = req.params;
+    const { return_message } = req.query;
+    if(return_message){
+        return res.status(200).send({msg: "Comment successfully deleted"})
+    }
```

Comment:
By returning here, you are not actually even reaching the selectCommentAndDelete logic so it never actually deleting anything